src/libstd/keyword_docs.rs

```diff
@@ -1100,10 +1100,25 @@ mod trait_keyword {}
 //
 /// A value of type [`bool`] representing logical **true**.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// Logically `true` is not equal to [`false`].
```

Comment:
```suggestion
/// Logically, `true` is unequal to [`false`].
```

Reply:
I liked `not equal` as it could be read in two ways and both are correct. true is not equal to false and true equals not false.